concepts/conditionals/about.md

```diff
@@ -1,48 +1,140 @@
+
+| Method | Description           | Example |
+| ------ | --------------------- | ------- |
+| <      | less than             | 5 < 4   |
+| <=     | less than or equal    | 4 <= 4  |
+| >      | greater than          | 3 > 1   |
+| >=     | greater than or equal | 2 >= 2  |
+
+The equal and not equal operators can be used to compare any type of value contrary to the operators already mentioned.
+The `==` operator is used to check if two values are equal, and that includes checking the type of the value.
+The `!=` works the same way but it will return `true` if the values are not equal and `false` if they are equal.
+Here is a list of the equal and not equal operators and an example of when they give a `true` value:
+
+| Method | Description  | Example |
+| ------ | ------------ | ------- |
+| ==     | equal        | 4 == 4  |
+| !=     | not equal    | 5 != 4  |
+
+## Combined comparison operator
+
+The combined comparison operator (sometimes called spaceship operator) is a special comparison operator.
+It is special in the sense that it doesn't return a truthy or falsey value but it returns a number.
+It is written as `<=>` and it is used to compare 2 values.
+It will return `1` if the left value is greater than the right value, `-1` if the left value is less than the right value, and `0` if the values are equal.
 
 ```ruby
-x = 5
+1 <=> 2 # => -1
+2 <=> 2 # => 0
+3 <=> 2 # => 1
+```
 
-if x == 5
-  # Execute logic if x equals 5
-elsif x > 7
-  # Execute logic if x greater than 7
-else
-  # Execute logic in all other cases
+## If statement
+
+The [`if`][if] statement is used to check if a given condition is truthy or falsey.
+If the condition is truthy then the code inside the if statement will run.
+An `if` statement ends with the `end` keyword.
+
+```ruby
+value = 1
+if value == 1
+  "1 is equal to 1"
+end
+# => "1 is equal to 1"
+
+if value > 2
+  "1 is greater than 2"
+end
+# => nil
+```
+
+## Unless statement
+
+The `unless`unless statement works very similarly to the `if` statement but it will run the code inside the `unless` statement if the condition is falsey.
+
+```ruby
+value = 1
+unless value == 1
+  "1 is not equal to 1"
 end
+# => nil
+
+unless value > 2
+  "1 is not greater than 2"
+end
+# => "1 is not greater than 2"
 ```
 
-Sometimes you want to execute a statement (or statements) if a condition is _not_ true, for situations like that, Ruby implements the `unless` keyword:
+## Else statement
+
+The `else` statement can be used in conjunction with the `if` and `unless` statements.
+The `else` statement will be executed if the `if` branch or the `unless` branch is not executed.
 
 ```ruby
-x = 4
-unless x == 5
-  # Execute logic if x does not equal 5
+value = 1
+if value == 1
+  "1 is equal to 1"
 else
-  # Execute logic if x == 5
+  "1 is not equal to 1"
 end
+# => "1 is equal to 1"
+
+unless value < 2
+  "1 is not greater than 2"
+else
+  "1 is greater than 2"
+end
```

Comment:
This is perhaps considered a poor example, as we should prefer to not use `else` with `unless`.  The indication is that the conditional statement is probably "backward" or "opposite of what is wanted".